src/rpc/substrate/getBlock.ts

```diff
@@ -0,0 +1,26 @@
+import ApiPromise from "@polkadot/api/promise";
+import {SignedBlock} from '@polkadot/types/interfaces/runtime';
+import { BlockHash } from '@polkadot/types/interfaces/rpc';
+
+async function _getBlock(blockHash: BlockHash|string, api: ApiPromise): Promise<SignedBlock> {
+  return api.rpc.chain.getBlock(blockHash);
+}
+
+interface GetBlockParams {
+  blockId?: number;
```

Comment:
this should be either string(hash or "latest") or number(slot). Maybe you can rename to blockTag if it makes more sense where blockTag is any possible block identification. You dont need additional hash param

Reply:
@mpetrunic is this number **(slot)** something specific or just number?